index.html

```diff
@@ -120,67 +135,130 @@ <h3 class="tittle3">Finish mobile section 5</h3>
                         <button class="sbutton">JavaScript</button>
                     </div>
 
-                    <div>
+                    
                         <button class="start1">See this project →</button>
-                    </div>
                 </div>
+            </div>
         </section>
 
-        <section id="greetings">
-            <h1>About</h1>
-            <p class="firstp">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Atque doloribus maiores,
-                rem excepturi et exercitationem commodi dolore, veritatis reprehenderit ullam quas, nemo obcaecati
-                perferendis quisquam sunt placeat praesentium? Nobis, nihil.</p>
-            <button class="start2">Get my CV</button>
-        </section>
+</div>
 
-        <section id="languages">
-            <h4>Languages</h4>
+<section id="projects-desktop"> 
 
-            <ul class="ul1">
-                <li>JavaScript(ES5/ES6)</li>
-                <li>Ruby</li>
-                <li>HTML</li>
-                <li>CSS</li>
+    <div class="div1"> <h2 class="deskt1">Projects</h2></div>
```

Comment:
- [ ] Great job. kindly align the below images well so they follow the figma design
yours
![Screenshot (575)_LI](https://user-images.githubusercontent.com/65298048/199895989-7cbe3c73-ab47-4476-b806-dd11d7969dfd.jpg)

figma design
![Screenshot (574)](https://user-images.githubusercontent.com/65298048/199896058-61104a45-0350-4f56-9353-4c614031a549.png)